### PR TITLE
docker file and common

### PR DESCRIPTION
**Pull request by markdoughten**
- adding style

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   web:
     build:
       context: ./app
-      dockerfile: Dockerfile.dev
```
Comment (mhadam) [inline docker-compose.yml]: So docker-compose.yml is just the default docker-compose, so I think we should drop this. Dockerfile.prod only gets run on the server.

**Comment by mhadam** [comment]
The rest looks fine though & I think just merging to main will deploy ok automatically

**Review by mhadam — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (markdoughten) [reply docker-compose.yml]: changed back to dev

**Comment by mhadam** [comment]
lgtm 👍

**Review by mhadam — APPROVED** [review]

**Pull request merged**